src/Core/AdminConsole/Models/Data/Organizations/OrganizationUsers/OrganizationUserOrganizationDetails.cs

```diff
@@ -8,6 +8,7 @@ public class OrganizationUserOrganizationDetails
 {
     public Guid OrganizationId { get; set; }
     public Guid? UserId { get; set; }
+    public Guid? OrganizationUserId { get; set; }
```

Comment:
I don't think this is nullable. `UserId` is only nullable because an orgUser may not be linked to a user yet. But it will always have an orgUser Id, which is a primary key for that table.

Same in the response models.

Reply:
That explains it! I've made it non-nullable.